jongbeom/app.js

```diff
@@ -0,0 +1 @@
+console.log('hi');
```

Comment:
```suggestion
console.log('hi');
```

요 부분은 왜 들어간 코드인가요? 없어도 될 것 같습니다